apps/examples/touchscreen/touchscreen_main.c

```diff
@@ -196,10 +229,14 @@ int touchscreen_main(int argc, char *argv[])
 	}
 
 	if (argc == 2) {
-		if (!strcmp(argv[1], "start")) {
+		if (!strncmp(argv[1], "start", 5)) {
```

Comment:
This will allow "touchscreen startaaa". Is this ok?

Reply:
The svace is detecting it. so i changed

Reply:
oh I missed updersdanding, i got your point, I changed len